queue_job/jobrunner/runner.py

```diff
@@ -386,8 +387,9 @@ def initialize_databases(self):
                 _logger.debug('queue_job is not installed for db %s', db_name)
             else:
                 self.db_by_name[db_name] = db
-                for job_data in db.select_jobs('state in %s', (NOT_DONE,)):
-                    self.channel_manager.notify(db_name, *job_data)
+                with db.select_jobs('state in %s', (NOT_DONE,)) as cr:
+                    for job_data in cr:
```

Comment:
```suggestion
                    for job_data in cr.fetchall():
```

Reply:
@guewen why? My whole goal here is to fetch piecewise from the server.

Reply:
You are right :)